server.js

```diff
@@ -170,6 +174,40 @@ async function createExpressApp() {
       })
   })
 
+
+expressApp.post('/createRoom', validateCreateRoomParams, async (req, res) => {
+  const { eventNotificationUri, maxPeersCount, roomId } = req.body;
+
+  // If roomId is not provided, generate a random one
+  const roomIdToUse = roomId || randomString({ length: 8, numeric: true, letters: true }).toLowerCase();
+  // Port of connection websocket
+  const port = process.env.MEDIASOUP_CLIENT_PROTOOPORT
+  // Host or Ip where try connect ws
+  const announcedIp = config.https.ingressHost
+  // Ws url to response
+  const wsUrl = `wss://${announcedIp}:${port}?roomId=${roomIdToUse}`
+
+  try {
+    // Create or retrieve an existing room
+    const createRoom = await getOrCreateRoom({ roomId: roomIdToUse, consumerReplicas: 0, maxPeersCount });
```

Comment:
What happens if `roomId` is specified and the Room is already created? I think it should throw an error, since the room already has an associated maxPeerCount and notification URI.

Also what does that `createRoom` variable mean? Is it used anywhere?

Reply:
I added a validation right before creating the room also removed the unused variable.